commit df8aa9a656c6642700dee2ff2e17471cafff8483
Author: Kieran Bingham <kieran.bingham@ideasonboard.com>
Date:   Mon Mar 18 17:14:51 2019 +0000

    vcsm: Fix makefile include on out-of-tree builds
    
    The vc_sm module tries to include the 'fs' directory from the
    $(srctree). $(srctree) is already provided by the build system, and
    causes the include path to be duplicated.
    
    With -Werror this fails to compile.
    
    Remove the unnecessary variable.
    
    Signed-off-by: Kieran Bingham <kieran.bingham@ideasonboard.com>

diff --git a/drivers/char/broadcom/vc_sm/Makefile b/drivers/char/broadcom/vc_sm/Makefile
index 19ce263bc273..20b7646fe4b6 100644
--- a/drivers/char/broadcom/vc_sm/Makefile
+++ b/drivers/char/broadcom/vc_sm/Makefile
@@ -1,5 +1,5 @@
 ccflags-$(CONFIG_BCM_VC_SM) += -Werror -Wall -Wstrict-prototypes -Wno-trigraphs -O2
-ccflags-$(CONFIG_BCM_VC_SM) += -I"drivers/staging/vc04_services" -I"drivers/staging/vc04_services/interface/vchi" -I"drivers/staging/vc04_services/interface/vchiq_arm" -I"$(srctree)/fs/"
+ccflags-$(CONFIG_BCM_VC_SM) += -I"drivers/staging/vc04_services" -I"drivers/staging/vc04_services/interface/vchi" -I"drivers/staging/vc04_services/interface/vchiq_arm" -I"fs"
 ccflags-$(CONFIG_BCM_VC_SM) += -DOS_ASSERT_FAILURE -D__STDC_VERSION=199901L -D__STDC_VERSION__=199901L -D__VCCOREVER__=0 -D__KERNEL__ -D__linux__
 
 obj-$(CONFIG_BCM_VC_SM) := vc-sm.o
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1373_df8aa9a656c6642700dee2ff2e17471cafff8483.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1373_df8aa9a656c6642700dee2ff2e17471cafff8483.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
